pypeit/calibrations.py

```diff
@@ -418,11 +431,19 @@ def get_dark(self):
         elif len(dark_files) == 0:
             self.msdark = None
         else:
-            # TODO: Should this include the bias?
-            # Build it
+            # TODO: If a bias has been constructed and it will be subtracted
```

Comment:
So we should talk this one through, but I think if a user is doing dark subtraction, that usually precludes bias subtraction. What is the point of subtracting a bias from science and a dark if you are then going to subtract those two images from each other, i.e. the bias just cancels out?

My opinion is that we should be over-scen subtracting science and dark (if dealing with CCDs) to remove the zero-point drift, and then simply subtracting dark from science to remove any spatial pattern.